iOS/docs/kickoff/README.md

```diff
@@ -11,112 +11,57 @@ The following will be needed to start an iOS project kickoff process:
 ![rename scheme](./resources/xcode-scheme-rename.gif)
 
-Folders and groups should also be renamed. Run the following on your console to fasten the process:
-
-`$ brew install rename ack`
-`$ find . -name 'BaseProject*' -print0 | xargs -0 rename -S 'BaseProject' 'ProjectName'`
-`$ ack --literal --files-with-matches 'BaseProject' | xargs sed -i '' 's/BaseProject/ProjectName/g'`
-
-Verify every `BaseProject` appearance has been renamed, running:
-
-`$ ack --literal 'BaseProject'`
-
-If this hasn't print out anything, you are good to go!
+Folders and groups should also be renamed. To do this you can use the `rename` script that is in the root of the project by running: `./rename` and following the steps.
 
 #### Build Scripts Configuration
 
-Renaming is over! Time to move on to configuring the [build scripts](https://github.com/guidomb/ios-scripts) file. To do this, edit the `script/.env` file, which should already contain these lines (among some others):
-
-```
-PROJECT_NAME=BaseProject
-XCODE_PROJECT=BaseProject.xcodeproj
-```
-
-Replace the values for `PROJECT_NAME` and `XCODE_PROJECT` with the following ones:
+Time to move on to install the build scripts that will bootstrap our project. To do this clone or download [this repository.](https://github.com/guidomb/ios-scripts)
 
-```
-PROJECT_NAME=ProjectName
-XCODE_PROJECT=ProjectName.xcodeproj
-```
-
-Check the value for `REQUIRED_XCODE_VERSION` matches the installed version. You can check it with the command `xcodebuild -version`.
-
-Replace the values for `REQUIRED_SWIFTLINT_VERSION` and `REQUIRED_CARTHAGE_VERSION` with the last release version for each of them.
+Before doing anything check in a terminal the value for `xcodebuild -version` and the latest versions of SwiftLint and Carthage.
 You can check them in [SwiftLint releases](https://github.com/realm/SwiftLint/releases) and [Carthage releases](https://github.com/Carthage/Carthage/releases).
 
-#### Include Fastlane
-
-Move to the project's root folder and download the [Fastlane](https://docs.fastlane.tools) content from the [Fastlane repository](https://github.com/Wolox/fastlane-mobile) by doing:
-
-`git clone --depth=1 https://github.com/Wolox/fastlane-mobile.git fastlane; rm -rf fastlane/.git`
-
-This copies the `fastlane-mobile` repository into `fastlane` folder, and removes the `.git` directory.
+Now run ` ./install <your project directory>` and follow the steps. Select the options you need for your project. If you are not sure you can use the default values the script provides (by pressing `return` when asked for a value) in almost all cases except for:
+The name of your project.
+When asked to create a `.env` file select `y`.
+When the script asks for xCode, Swiftlint and Carthage versions use the values you searched for above.
+When asked if you want to bootstrap the project select `y`.
```

Comment:
should we add this too?

Reply:
You mean skipping that choice entirely?

Reply:
no, just set it as default so you can press `return` if you don't know what to choose

Reply:
I'm not sure about that, I think its ok that the default its just the installation. I mean, the script is called `install`. What we can do is provide a config file to set the defaults, but I'm not sure if thats worth it

Reply:
yeah, probably not worth it, leave it like this

Reply:
now that I look at it, `yes` is actually the default

Reply:
then we should remove this line from the readme